PULL_REQUEST_TEMPLATE.md

```diff
@@ -41,8 +35,17 @@ forms of testing, please use your best judgement or ask for guidance
 if you are unsure. A good rule of thumb is the riskier the change, the
 more comprehensive the testing should be.
 
-[optional *Deprecated*:]
-> Description of what is deprecated.
+[Optional Fixes #Issue]
+
+[Optional *API Changes*:]
+>Link to [Data Plane PR](https://github.com/envoyproxy/data-plane-api/pulls)]
+
+[Optional *Docs Changes*:]
+>Link to [Data Plane PR](https://github.com/envoyproxy/data-plane-api/pulls)]
+
+[Optional *Deprecated*:]
+>Description of what is deprecated.
 
-[optional *Release Notes*:] If this change is user impacting you **must** add a release note to
+[Optional *Release Notes*:]
```

Comment:
Can you move this to the first optional thing to make sure people look at it?

Reply:
I would be in favor of making this mandatory (enforced by a presubmit check) and having folks fill in `{n/a, na, n.a.}` etc. when there is no impact just to ensure that everyone gives this some thought.

Reply:
I agree, but I don't think we should block this PR on that (we should update the text though to be required and have the user specify one of the options). I opened https://github.com/envoyproxy/envoy/issues/2133 to track automation.

Reply:
I moved to mandatory section to force people to think about it.

Reply:
I don't see any changes / DCO is broken. Can you fix?

Reply:
I committed through the github UI, forgot about the DCO. Fixed now.